generators/client/templates/src/main/webapp/app/admin/health/_health.modal.controller.js

```diff
@@ -17,5 +17,14 @@
         function cancel() {
             $uibModalInstance.dismiss('cancel');
         }
+
```

Comment:
Oh ha ha ya he really does have a strange user name.

Thanks & regards,
Deepu

On 6 Sep 2016 19:29, "Julien Dubois" <notifications@github.com> wrote:

> In generators/client/templates/src/main/webapp/app/admin/
> health/_health.modal.controller.js
> <https://github.com/jhipster/generator-jhipster/pull/4078#discussion_r77639586>
> :
>
> > @@ -17,5 +17,14 @@
> >          function cancel() {
> >              $uibModalInstance.dismiss('cancel');
> >          }
> > +
>
> Oh sorry @deepu105 <https://github.com/deepu105> I should have introduced
> you to @040796 <https://github.com/040796> (who definitely has a strange
> username) - he's my new trainee, I'm going to have a look at this with him
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/4078/files/c59aab167619b2841dbc7d526af79c433482d5b3#r77639586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDlF3evmgVotzdkzsjavUJ3QQZSAcFvks5qnXHSgaJpZM4J1syC>
> .
>